_search-plugins/sql/functions.md

```diff
@@ -18,166 +18,170 @@ The SQL plugin supports the following common functions shared across the SQL and
+| day_of_year\*      | `day_of_year(date) -> integer`                                                         | `SELECT day_of_year(date('2020-08-26'))`                                            |
+| extract\*          | `extract(part FROM date) -> integer`                                                   | `SELECT extract(MONTH FROM datetime('2020-08-26 10:11:12'))`                        |
+| from_days          | `from_days(N) -> integer`                                                              | `SELECT from_days(733687)`                                                          |
+| from_unixtime      | `from_unixtime(N) -> date`                                                             | `SELECT from_unixtime(1220249547)`                                                  |
+| get_format         | `get_format(PART, string) -> string`                                                   | `SELECT get_format(DATE, 'USA')`                                                    |
+| hour               | `hour(time) -> integer`                                                                | `SELECT hour((time '01:02:03'))`                                                    |
+| hour_of_day\*      | `hour_of_day(time) -> integer`                                                         | `SELECT hour_of_day((time '01:02:03'))`                                             |
+| last_day\*         | `last_day(date) -> integer`                                                            | `SELECT last_day(date('2020-08-26'))`                                               |
+| localtime          | `localtime() -> date`                                                                  | `SELECT localtime()`                                                                |
+| localtimestamp     | `localtimestamp() -> date`                                                             | `SELECT localtimestamp()`                                                           |
+| makedate           | `makedate(double, double) -> date`                                                     | `SELECT makedate(1945, 5.9)`                                                        |
+| maketime           | `maketime(integer, integer, integer) -> date`                                          | `SELECT maketime(1, 2, 3)`                                                          |
+| microsecond        | `microsecond(expr) -> integer`                                                         | `SELECT microsecond((time '01:02:03.123456'))`                                      |
+| minute             | `minute(expr) -> integer`                                                              | `SELECT minute((time '01:02:03'))`                                                  |
+| minute_of_day\*    | `minute_of_day(expr) -> integer`                                                       | `SELECT minute_of_day((time '01:02:03'))`                                           |
```

Comment:
```suggestion
| minute_of_day\*    | `minute_of_day(expr) -> integer`                                                       | `SELECT minute_of_day(time '01:02:03')`                                           |
```

Reply:
fixed in https://github.com/opensearch-project/documentation-website/pull/4232/commits/9442d3b1e8ec65a7c0e9b166073cf2b8281553d1